src/main.cxx

```diff
@@ -0,0 +1,30 @@
+#include "cl.h"
+#include "winrpath.h"
+#include "utils.h"
+
+
+int main(int argc, char* argv[]) {
+
+    if (isPatch(argv, argv+argc)) {
+        StrList patch_args = parsePatch(argv);
+        LibRename patcher(patch_args[0], patch_args[1], true);
+        patcher.computeDefFile();
+        patcher.executeLibRename();
+    }
```

Comment:
Broadly speaking, my understanding is that this updates the `cl` wrapper so that if the word "patch" appears as an argument, it is capable of performing relocation. I don't see the modification of non-relocation toolchain executions though. Also, that seems like a loose check: I think that token could appear in the invocation in other ways.

Reply:
This has been refactored fairly substantially, I believe the refactor addresses all concerns raised here.